source/content/private-paths.md

```diff
@@ -3,31 +3,44 @@ title: Private Paths for Files and Code
 
 Determining which path to use depends on whether or not the data should be tracked with Git as part of your site's codebase. For example, secret tokens or credentials for third party services should not be version controlled alongside your site's code.
 
-**Private Path for Files (Not Version Controlled)**
+#### Private Path for Files (Not Version Controlled)
+
 Drupal: `sites/default/files/private`
 WordPress: `wp-content/uploads/private`
 
-**Private Path for Code (Version Controlled)**
+#### Private Path for Code (Version Controlled)
+
 Drupal and WordPress: `private`
 
 <Alert title="Note" type="info">
+
 If you have not already created these directories, you will need to do that first. Create the folders in Dev via SFTP or Git, and push the changes to your Test and Live environments.
+
 </Alert>
 
 ## Private Path for Code
+
 Store data that should be version controlled, such as [Quicksilver](/quicksilver) scripts, within the `private` directory at the root level of your site's codebase (same level as `index.php`). If you're connecting via SFTP, navigate into the `code` directory and upload files to `private`. If you're connecting via Git, use the `private` directory at the root level of your cloned repository. The private path for code is the same for both Drupal and WordPress sites.
 
 ## Private Path for Files
+
 When it comes to keeping production keys secure, the best solution is to use a key management service like [Lockr](https://lockr.io/) to automatically encrypt and secure keys on distributed platforms such as Pantheon. You can integrate this service using the [Lockr plugin](https://wordpress.org/plugins/lockr/) for WordPress and the [Lockr module](https://www.drupal.org/project/lockr) for Drupal. For more details, see this [related blog post](https://pantheon.io/blog/key-drupal-security).
 
-Alternatively, you can store sensitive data in a JSON or ini-style text file within the `wp-content/uploads/private` (WordPress) or `sites/default/files/private` (Drupal) directories. These directories are symbolically linked to Valhalla and can also be accessed from the `files` directory when connecting via SFTP. This allows secure data to be distributed to other environments, while keeping it out of version control. You can then read the data from `settings.php` or `wp-config.php`, like so:
+Alternatively, you can store sensitive data in a JSON or ini-style text file within the following directories:
 
-```php
+- `wp-content/uploads/private` (WordPress)
+- `/wp-content/uploads/private/sites/blog_id/` (WordPress Multisite)
+- `sites/default/files/private` (Drupal)
+
+These directories are symbolically linked to Valhalla and can also be accessed from the `files` directory when connecting via SFTP. This allows secure data to be distributed to other environments, while keeping it out of version control. You can then read the data from `settings.php` or `wp-config.php`, like so:
```

Comment:
For the technical review, this is the relevant change. The rest is copy updates.

Reply:
Looks good and read well.